core/modules/views/tests/views_layout.test

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * @file
+ * Tests for views and layout collab
+ * @property \stdClass $admin_user
+ */
+
+require_once BACKDROP_ROOT . '/core/modules/views/tests/views_query.test';
+
+/**
+ * Tests the interface for adding, removing, and moving blocks.
+ */
+class ViewsLayoutsTest extends BackdropWebTestCase {
+
+  /**
+   * @inheritdoc
+   */
+  protected function setup() {
+    // Loading in required modules.
+    parent::setup(
+      'views', 'views_ui', 'layout', 'entity_ui', 'block', 'user', 'contextual'
+    );
+  }
+
+  /**
+   * Tests that a contextual filter input can be passed in from a layout.
+   */
+  public function testViewsLayouts() {
+    global $admin_user;
+
+    $permissions = array(
+      'administer site configuration',
+      'administer views',
+      'administer blocks',
+      'bypass node access',
+      'access user profiles',
+      'view revisions',
+      'synchronize configuration',
+      'administer layouts',
+      'administer flexible templates',
+    );
+
+    // Create and login as an admin with views and layouts permissions.
+    $admin_user = $this->backdropCreateUser($permissions);
+    $this->backdropLogin($admin_user);
+
+    // Navigate to the form that allows the adding of our view.
+    $this->backdropGet('admin/structure/views/add');
+
+    // <editor-fold desc="Create our layout context view and configure its
```

Comment:
@rbargerhuff What is `<editor-fold>`? If this is specific to an IDE I don't think we should be keeping that in the codebase.

Reply:
So `<editor-fold>` along with _region_ comments are specific to IDEs and proper programming Text-Editors. They allow the coder to define regions of text that can be collapsed. I would hope we would support these seeing how there is an effort to improve the readability of code. Perhaps there should be a discussion about this with the community?